### PR TITLE
Known issue for vSphere IPI upgrades

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by codyhoag** [comment]
@jinyunma PTAL. Thanks!

**Comment by jinyunma** [comment]
It's better to reword the description and add more details, something like below:
When upgrading a cluster from version 4.5 to 4.6 that was installed with installer-provisioned infrastructure on vSphere, in case control plane nodes' IPs are changed during upgrading which lead to upgrade failed, please reserve them before upgrading to 4.6. You can do this on the DHCP server by adding the IPs into the `/etc/dhcp/dhcpd.conf` file and then restarting dhcpd process.

**Comment by codyhoag** [comment]
@jcpowermac what are your thoughts on adding more details, as @jinyunma suggested? I want to make sure I'm considering the points you brought up yesterday. How about something like this:
```
When upgrading a cluster from version 4.5 to 4.6 that was installed with installer-provisioned
infrastructure on vSphere, the upgrade fails if the control plane node IPs change during the
upgrade. As a workaround, you must reserve the control plane node IPs before upgrading to
version 4.6. You can do this by adding the IPs into the `/etc/dhcp/dhcpd.conf` file on the DHCP
server and then restarting the DHCP daemon process.
```

**Comment by jcpowermac** [comment]
> You can do this by adding the IPs into the `/etc/dhcp/dhcpd.conf` file on the DHCP
> server and then restarting the DHCP daemon process.

If it can be less Linux (RHEL) specific. We don't know which DHCP server the customer is using.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -1465,6 +1465,13 @@ When bypassing this check, ensure that the credentials you provide have the xref
 
 * When deploying an egress router pod in DNS proxy mode, the pod fails to initialize. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888024[*BZ#1888024*])
 
+* When upgrading a cluster from version 4.5 to 4.6 that was installed with
+installer-provisioned infrastructure on vSphere, the upgrade fails if the
+control plane node IPs change during the upgrade. As a workaround, you must
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:1470]:
```suggestion
control plane node IP addresses change during the upgrade. As a workaround, you must
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -1465,6 +1465,13 @@ When bypassing this check, ensure that the credentials you provide have the xref
...
+reserve the control plane node IPs before upgrading to version 4.6. You can do
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:1471]:
```suggestion
reserve the control plane node IP addresses before upgrading to version 4.6.
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -1465,6 +1465,13 @@ When bypassing this check, ensure that the credentials you provide have the xref
...
+this by reviewing your DHCP server's documentation for the configuration of
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:1472]:
```suggestion
Review your DHCP server's documentation for the configuration of
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -1465,6 +1465,13 @@ When bypassing this check, ensure that the credentials you provide have the xref
 
 * When deploying an egress router pod in DNS proxy mode, the pod fails to initialize. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888024[*BZ#1888024*])
 
+* When upgrading a cluster from version 4.5 to 4.6 that was installed with
+installer-provisioned infrastructure on vSphere, the upgrade fails if the
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:1469]: Double-check that the first instance of vSphere is "VMware vSphere."

**Pull request merged**